src/store/audio.ts

```diff
@@ -1787,79 +1787,99 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
     ),
   },
 
+  SET_AUDIO_SOURCE: {
+    mutation(_, { audioBlob }: { audioBlob: Blob }) {
+      getAudioElement().src = URL.createObjectURL(audioBlob);
+    },
+  },
+
   PLAY_AUDIO_BLOB: {
     action: createUILockAction(
       async (
         { state, commit, dispatch },
         { audioBlob, audioKey }: { audioBlob: Blob; audioKey?: AudioKey }
       ) => {
-        getAudioElement().src = URL.createObjectURL(audioBlob);
+        commit("SET_AUDIO_SOURCE", { audioBlob });
+        let offset: number | undefined;
         // 途中再生用の処理
         if (audioKey) {
           const accentPhraseOffsets = await dispatch("GET_AUDIO_PLAY_OFFSETS", {
             audioKey,
           });
-          if (accentPhraseOffsets.length === 0) {
-            getAudioElement().currentTime = 0;
-          } else {
-            const startTime =
-              accentPhraseOffsets[state.audioPlayStartPoint ?? 0];
-            if (startTime === undefined) throw Error("startTime === undefined");
-            // 小さい値が切り捨てられることでフォーカスされるアクセントフレーズが一瞬元に戻るので、
-            // 再生に影響のない程度かつ切り捨てられない値を加算する
-            getAudioElement().currentTime = startTime + 10e-6;
-          }
+          if (accentPhraseOffsets.length === 0)
+            throw new Error("accentPhraseOffsets.length === 0");
+          const startTime = accentPhraseOffsets[state.audioPlayStartPoint ?? 0];
+          if (startTime === undefined) throw Error("startTime === undefined");
+          // 小さい値が切り捨てられることでフォーカスされるアクセントフレーズが一瞬元に戻るので、
+          // 再生に影響のない程度かつ切り捨てられない値を加算する
+          offset = startTime + 10e-6;
         }
 
-        // 一部ブラウザではsetSinkIdが実装されていないので、その環境では無視する
-        if (getAudioElement().setSinkId) {
-          getAudioElement()
-            .setSinkId(state.savingSetting.audioOutputDevice)
-            .catch((err) => {
-              const stop = () => {
-                getAudioElement().pause();
-                getAudioElement().removeEventListener("canplay", stop);
-              };
-              getAudioElement().addEventListener("canplay", stop);
-              window.electron.showMessageDialog({
-                type: "error",
-                title: "エラー",
-                message: "再生デバイスが見つかりません",
-              });
-              throw new Error(err);
+        return dispatch("PLAY_AUDIO_PLAYER", { offset, audioKey });
+      }
+    ),
+  },
+
+  PLAY_AUDIO_PLAYER: {
```

Comment:
すごく細かいですが、次のリファクタリングが来るまでにforkした方にも文脈がわかるよう、この関数がなぜ切り出されているかコメントしておくのどうでしょう。
こんな感じ･･･？
```suggestion
  // NOTE: リファクタリング中、別ファイルに移動予定
  // https://github.com/VOICEVOX/voicevox/pull/1553
  PLAY_AUDIO_PLAYER: {
```

`SET_AUDIO_SOURCE`もでしたっけ、ちょっと忘れました。。